gsy_framework/sim_results/electric_blue/aggregate_results.py

```diff
@@ -10,9 +10,31 @@
 # Used by forward markets; the following dictionary defines
 # what aggregations are needed for each market type.
 MARKET_RESOLUTIONS = {
```

Comment:
Does this need to be in gsy-framework, or can we move it to gsy-e?

Reply:
gsy-results will make use of it for its implementation. If we move it to gsy-e, then gsy-results can't access it!